iroh-store/src/main.rs

```diff
@@ -31,45 +25,24 @@ async fn main() -> eyre::Result<()> {
     println!("Starting iroh-store, version {version}");
 
     let args = Args::parse();
-    let config = Config {
-        path: args.path.clone(),
-    };
+    let config = Config::new(args.path.clone());
+    let rpc_addr = config.rpc.store_addr;
 
-    let _store = if config.path.exists() {
+    let store = if config.path.exists() {
         info!("Opening store at {}", config.path.display());
         Store::open(config).await?
     } else {
         info!("Creating store at {}", config.path.display());
         Store::create(config).await?
     };
 
+    tokio::spawn(async move {
```

Comment:
please abort the resulting handle, after receiving the sigint, for a clean shutdown